lua/entities/gmod_wire_teleporter.lua

```diff
@@ -169,7 +169,12 @@ function ENT:Jump_Part2( withangles )
 	self.LocalPos = {}
 	self.LocalAng = {}
 	self.LocalVel = {}
-	for _, ent in pairs( self.Entities ) do
+	for k, ent in pairs( self.Entities ) do
+		if not IsValid(ent) then
+			self.Entities[k] = nil
+			continue
```

Comment:
Change the continue to a goto or invert the if statement and wrap the entire block?

Reply:
What's wrong with continue? it's used in various places throughout wiremod

Reply:
Just part of the style guide. Those parts are outdated.